TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/test_cases/backfill_tests.py

```diff
@@ -73,3 +95,146 @@ def backfill_wait_for_stop(self):
                                                       delay=5,
                                                       max_attempts=30)
         return super().backfill_wait_for_stop()
+
+
+class Test0007EndToEndTestForES8WithOSBenchmarks(MATestBase):
+    def __init__(self, console_config_path: str, console_link_env: Environment, unique_id: str):
+        allow_combinations = [
+            (ElasticsearchV8_X, OpensearchV2_X)
```

Comment:
Can we avoid creating a new test and add this matrix value onto an existing test case?